assets/src/styles/blocks/CarouselHeader/CarouselHeaderStyle.scss

```diff
@@ -549,12 +559,16 @@ $medium-image-height: 600px;
         display: block;
         margin: auto;
       }
+
+      html[dir="rtl"] & {
+        transform: rotate(180deg);
```

Comment:
Should work, but we should be mindful of possible adverse affects of doing a CSS transform. Doing a rotate would not only swap left and right, but also up and down (not sure if that results in any visual or UX difference, but it might not play well with paddings). Maybe mirroring it using `transform: scaleX(-1)` would be a bit safer?

Reply:
Updated! Also added a small fix for the transitions on hover 👍